src/config.template.cjs

```diff
@@ -122,9 +124,10 @@ module.exports = {
 
 	/**
 	 * cross-seed will use links of this type to inject data-based matches into your client.
+	 * https://www.cross-seed.org/docs/basics/faq-troubleshooting#what-linktype-should-i-use
 	 * Options: "symlink", "hardlink"
 	 */
-	linkType: "symlink",
+	linkType: "hardlink",
```

Comment:
This would seem to break things when trying to hardlink to differing partition/disk/mount than what is set in `linkDir:`, meaning OOTB, this would seem to break things for those who have media spread.

Would seem to require https://github.com/cross-seed/cross-seed/issues/625 first.

Reply:
I think at this point, my impression was that we are more opting for these defaults to be recommendations and we are recommending people utilize hardlinks.

These are not necessarily going to work for everyone, all the time, and will always require some level of adjusting or configuration.

I'm not sure having settings that automatically work for everything OOTB because its more or less "inferior" or whatever you want to call it is a route I would opt for...but I'm open for discussions.

Reply:
Given the issues I've seen people experience regarding hardlinks and arr's, I'm not convinced this is the appropriate default setting. In my mind, this has more potential for endless complaints than setting for symlinks.

At the end of the day, people *should* research and pick the appropriate setting, but unfortunately, people often don't.

It basically boils down to:
Does the desire to strongly point people to hardlinks, outweigh the amount of bitching likely to be experienced.

Reply:
I understand the potential consequences...perhaps we leave it set to "DIDNT_READ_CONFIG" and link to the docs :P

Reply:
I think hard links and symlinks both have their own problems, but I'm inclined to listen to the people that have the most experience diagnosing and helping people with issues, which are @zakkarry and @bakerboy448, who have both advocated for hardlinks. If we switch the other way and suddenly see a massive uptick in support requests for hardlink issues, we can always change the recommendation back.